nitransforms/io/afni.py

```diff
@@ -119,5 +124,29 @@ def from_string(cls, string):
         return _self
 
 
+class AFNIDisplacementsField(DisplacementsField):
+    """A data structure representing displacements fields."""
+
+    @classmethod
+    def from_image(cls, imgobj):
+        """Import a displacements field from a NIfTI file."""
+        _hdr = imgobj.header.copy()
+        _shape = _hdr.get_data_shape()
+
+        if (
+            len(_shape) != 5 or
+            _shape[-2] != 1 or
+            not _shape[-1] in (2, 3)
+        ):
+            raise TransformFileError(
+                'Displacements field "%s" does not come from AFNI.' %
+                imgobj.file_map['image'].filename)
```

Comment:
if you can add a test to cover these lines (so we don't lose any coverage), we should be good to go!

Reply:
@smoia - any chance you can get smoia/nitransforms#1 merged anytime soon?